tests/test_govuk_frontend.py

```diff
@@ -745,6 +798,42 @@ def test_from_question_with_errors(self, question, snapshot):
 
         assert form == snapshot
 
+    def test_from_question_with_followup(self, question_with_followup, snapshot):
+        assert from_question(question_with_followup) == snapshot
+
+    def test_from_question_with_followup_with_data(self, question_with_followup, snapshot):
+        data = {
+            "yesNo": "True",
+            "tellUsMore": "Gosh, there's a lot to say.",
+        }
+
+        form = from_question(question_with_followup, data)
+
+        assert form[1]["params"]["items"][0]["checked"]
+        assert (
+            form[1]["params"]["items"][0]["conditional"]["html"]["params"]["value"]
+            == "Gosh, there's a lot to say."
+        )
+
+        assert form == snapshot
+
+    def test_from_question_with_followup_with_errors(self, question_with_followup, snapshot):
+        errors = {
+            "tellUsMore": {
+                "input_name": "tellUsMore",
+                "href": "#input-answer",
+                "question": "Tell us more",
+                "message": "Enter an answer.",
+            },
+        }
+
+        form = from_question(question_with_followup, errors=errors)
+
+        assert "errorMessage" not in form[1]["params"]
+        assert form[1]["params"]["items"][0]["conditional"]["html"]["params"]["errorMessage"]
```

Comment:
Should we assert the expected error message text?

Reply:
Eh, I don't think it matters too much what the error message text is, just that it's there. In any case the snapshot assertion below will catch any situations where the message text is mangled; a representation of the value gets compared to the saved snapshot in https://github.com/alphagov/digitalmarketplace-content-loader/pull/132/files#diff-e9a82f20271c1113cecfad9c08777a4a380a669f4ed9a5b71c840323bec42125R581.